lsst/ts/tunablelaser/csc.py

```diff
@@ -49,38 +49,37 @@ class LaserErrorCode(enum.IntEnum):
     hw_cpu_error = 7304
 
 
-class LaserCSC(salobj.BaseCsc):
+class LaserCSC(salobj.ConfigurableCsc):
     """This is the class that implements the TunableLaser CSC.
 
     Parameters
     ----------
-    address: str
+    address : `str`
         The physical usb port string where the laser is located.
-    frequency: float, optional
+    frequency : `float`, optional
         The amount of time that the telemetry stream is published.
-    initial_state: salobj.State, optional
+    initial_state : `salobj.State`, optional
         The initial state that a CSC will start up in. Only useful for unit tests as most CSCs
-        will start in `State.STANDBY`
+        will start in `salobj.State.STANDBY`
 
     Attributes
     ----------
-    model: LaserModel
+    model : `LaserModel`
         The model of the laser component which handles the actual hardware.
-    frequency: float
+    frequency : `float`
         The amount of time that telemetry waits to publish.
     wavelength_topic
     temperature_topic
 
     """
-    def __init__(self, address, configuration, frequency=1, initial_state=salobj.State.STANDBY):
-        super().__init__(SALPY_TunableLaser)
+    def __init__(self,frequency=1, initial_state=salobj.State.STANDBY,index=None,schema="/home/ecoughlin/gitrepo/ts_laser/schema/TunableLaser.yaml"):
```

Comment:
The default schema is pointing to a file in your user computer. Probably not a good idea. I would suggest you use `schema=None` as the default and, if that is the case, you unpack a relative path.